packages/appstore/src/components/currency-switcher-card/real/real-account-card.tsx

```diff
@@ -31,11 +31,7 @@ const RealAccountCard = observer(() => {
     return (
         <CurrencySwitcherContainer
             className='demo-account-card'
-            title={
-                <Text size='xs' line_height='s'>
-                    {getCurrencyName(currency)}
-                </Text>
-            }
+            title={<BalanceText currency={get_currency} balance={formatMoney(currency, balance, true)} size='xs' />}
```

Comment:
@arshad-rao-deriv In my opinion, we don't need to write `balance={formatMoney(currency, balance, true)}` because balance will be converted the same way inside of `BalanceText` component. I understand that it was, but we can improve the code.